darts/explainability/explainability_result.py

```diff
@@ -16,6 +18,45 @@
 logger = get_logger(__name__)
 
 
+def check_input_validity_for_querying_explainability_result(func: Callable) -> Callable:
```

Comment:
Could we simply make this a normal function and call it at the begging in each method?

Wrappers can swallow up the actual method/function arguments in the documentation (from IDE for example) without some additional code